src/wifipaf/WiFiPAFEndPoint.cpp

```diff
@@ -797,78 +787,88 @@ SequenceNumber_t WiFiPAFEndPoint::AdjustRemoteReceiveWindow(SequenceNumber_t las
     return static_cast<uint8_t>(newRemoteWindowBoundary - newestUnackedSentSeqNum);
 }
 
-CHIP_ERROR WiFiPAFEndPoint::GetPktSn(Encoding::LittleEndian::Reader & reader, uint8_t * pHead, SequenceNumber_t & seqNum)
+CHIP_ERROR WiFiPAFEndPoint::GetPktSn(Encoding::LittleEndian::Reader & reader, SequenceNumber_t & seqNum)
```

Comment:
We pass in a non-const reference to a reader - why? Do we expect it to change? We seem to go through extra steps to ensure that the reader does not change content/position.

Should this be a const reference instead?

Reply:
Not passing a reference should be better. I will change it